lib/shortcodes/county-contact-info/county-contact-shortcode.php

```diff
@@ -0,0 +1,221 @@
+		'map_address'  => '',
+		'content'      => '',
+	);
+
+
+	public function __construct() {
+
+		$core_setting = get_theme_mod( 'extension_core_county_contact', '' );
+
+		if ( ! empty( $core_setting ) ) {
+
+			\add_action( 'init', array( $this, 'register_shortcode' ) );
+
+			\add_filter( 'cpb_shortcodes', array( $this, 'register_cpb_shortcode' ) );
+
+			\add_filter( 'cpb_to_shortcode', array( $this, 'cpb_to_shortcode' ) );
+
+			\add_filter( 'cpb_clean_atts', array( $this, 'cpb_clean_atts' ), 10, 3 );
+
+		} // End if
+
+	} // End __construct
+
+
+	/*
+	* @desc Register section shortcode
+	* @since 3.0.0
+	*/
+	public function register_shortcode() {
+
+		\add_shortcode( 'county_contact_info', array( $this, 'get_rendered_shortcode' ) );
+
+	} // End register_shortcode
+
+
+	/*
+	* @desc Register Shortcode with Pagebuilder
+	* @since 0.0.1
+	*/
+	public function register_cpb_shortcode( $shortcodes ) {
+
+		$default_atts = apply_filters( 'cpb_shortcode_default_atts', $this->default_settings, array(), 'county_contact_info' );
+
+		$shortcodes['county_contact_info'] = array(
+			'form_callback'         => array( $this, 'get_shortcode_form' ),
+			'label'                 => 'County Contact Info', // Label of the item
+			'render_callback'       => array( $this, 'get_rendered_shortcode' ), // Callback to render shortcode
+			'default_atts'          => $default_atts,
+			'in_column'             => true, // Allow in column
+		);
+
+		return $shortcodes;
+
+	} // End register_cpb_shortcode
+
+
+	/*
+	* @desc Render the shortcode
+	* @since 3.0.0
+	*
+	* @param array $atts Shortcode attributes
+	* @param string $content Shortcode content
+	*
+	* @return string HTML shortcode output
+	*/
+	public function get_rendered_shortcode( $atts, $content ) {
+
+		$html = '';
+
+		$default_atts = apply_filters( 'cpb_shortcode_default_atts', $this->default_settings, $atts, 'county_contact_info' );
+
+		// Check default settings
+		$atts = \shortcode_atts( $default_atts, $atts, 'county_site_search_form' );
+
+		// Let's just check and make sure this exists first
+		if ( function_exists( 'spine_get_option' ) ) {
+
+			$department = spine_get_option( 'contact_department' );
+
+			$address = spine_get_option( 'contact_streetAddress' );
+
+			$locality = spine_get_option( 'contact_addressLocality' );
+
+			$zip_code = spine_get_option( 'contact_postalCode' );
+
+			$telephone = spine_get_option( 'contact_telephone' );
+
+			$email = spine_get_option( 'contact_email' );
+
+			$contact_point = spine_get_option( 'contact_ContactPoint' );
+
+			$contact_title = spine_get_option( 'contact_ContactPointTitle' );
+
+			if ( ! empty( $atts['show_map'] ) ) {
+
+				wp_enqueue_script( 'google_maps_api', '//maps.googleapis.com/maps/api/js', array(), false, true );
+
+				wp_enqueue_script( 'google-map-embed', ecore_get_plugin_url( 'lib/js/google-map-embed.js' ), array( 'google_maps_api' ), false, true );
+
+				$map_address = $address . ' ' . $locality . ' ' . $zip_code;
```

Comment:
Everything retrieved from `spine_get_option()` should be considered untrusted and needs to be escaped on output.